test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -8361,7 +8361,25 @@ def cleanup_jobs(self, extend=None, runas=None):
             delete_xt += 'deletehist'
             select_xt = 'x'
         job_ids = self.select(extend=select_xt)
-        if len(job_ids) > 0:
+
+        # Make sure the current user is a manager. Some tests might have
+        # unset the mangers attribute. Ignore 'Duplicate entry in the list'
+        # error if the current user was already a manager
+        current_user = pwd.getpwuid(os.getuid())[0]
```

Comment:
if user passes runas arguments then don't we need to add that user to manager list

Reply:
runas argument is not used for any qmgr commands in this function. So it may not be needed to add that user to the manager list